Task_1_4_1/src/main/java/RecordBook.java

```diff
@@ -0,0 +1,153 @@
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class that implements the work of the student's electronic record book.
+ */
+public class RecordBook {
+    private Integer amountOfAllGrades;
+    private Integer sumOfAllGrades;
+    private Integer amountOfThrees;
+    private HashMap<String, ArrayList<Integer>> finalGrades;
+    private HashMap<Integer, HashMap<String, Integer>> grades;
+
+    /**
+     * Constructor.
+     */
+    public RecordBook() {
+        this.amountOfAllGrades = 0;
+        this.sumOfAllGrades = 0;
+        this.amountOfThrees = 0;
+        this.grades = new HashMap<>();
+        for (int i = 0; i < 8; i++) {
+            this.grades.put(i + 1, new HashMap<>());
+        }
+        this.finalGrades = new HashMap<>();
+    }
+
+    /**
+     * Converting a grade from a string to a number.
+     *
+     * @param grade Grade as string
+     * @return Grade as number
+     */
+    private int convertGrade(String grade) {
+        if (grade.compareTo("удовлетворительно") == 0) {
+            return 3;
+        } else if (grade.compareTo("хорошо") == 0) {
+            return 4;
+        } else if (grade.compareTo("отлично") == 0) {
+            return 5;
+        } else {
+            return 0;
+        }
```

Comment:
1. В Java 14 появились такие удобные switch операторы, можно возспользоваться
2. Возвращать какое-либо значение в качестве некорректного считается плохой практикой, поскольку это ухудшает читаемость и универсальность программы (вдруг когда-то мы захотим добавить оценку 0). Для этих целей как раз существуют исключения.
Я предлагаю попробовать такой вариант 
```suggestion
return switch (grade) {
            case "удовлетворительно" -> 3;
            case "хорошо" -> 4;
            case "отлично" -> 5;
            default -> throw new IllegalArgumentException();
        };
```

Reply:
Почему-то gradle build ругается на такую запись, хотя в intellij idea всё отрабатывает нормально. Ещё я столкнулась с такой проблемой, что локально у меня проходят все тесты, но во всех предыдущих коммитах в логах написано, что успешно прошел только 1 из 4 тестов